src/v3/src/components/Form/style.module.css

```diff
@@ -0,0 +1,4 @@
+:global(span.strong) {
+  font-weight: bold;
+  word-break: break-all;
+}
```

Comment:
Curious if this can be handled in the `ScopedCssBaseline` component: https://mui.com/material-ui/react-css-baseline/#typography

Reply:
Totally forgot about this 😅